nixos/modules/services/databases/openldap.nix

```diff
@@ -237,8 +237,8 @@ in
   config = mkIf cfg.enable {
     assertions = [
       {
-        assertion = cfg.rootpwFile != null || cfg.rootpw != null;
-        message = "Either services.openldap.rootpw or services.openldap.rootpwFile must be set";
+        assertion = cfg.configDir != null || cfg.rootpwFile != null || cfg.rootpw != null;
```

Comment:
Can be written as `cfg.configDir == null -> cfg.rootpwFile != null || cfg.rootpw != null` for more clarity, but that's completely not important.

Reply:
Thanks for reviewing! I think that was how I wrote it originally and I found it more confusing that way. Maybe it's just confusing no matter how it's written. :sweat: